File: dataset/output-bk/PERPLEXITY_llama-3-sonar-large-32k-chat/Task73_PERPLEXITY_llama-3-sonar-large-32k-chat.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter password:");
        String password = scanner.nextLine();
        if (password.equals("admin123")) {
            System.out.println("Password is valid");
        } else {
            System.out.println("Password is not valid");
        }
    }
}

File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task73_PERPLEXITY_llama-3-sonar-large-32k-chat.c
#include <stdio.h>
#include <string.h>

int main() {
    char password[20];
    printf("Enter password:");
    scanf("%19s", password);
    if (strcmp(password, "admin123") == 0) {
        printf("Password is valid
");
    } else {
        printf("Password is not valid
");
    }
    return 0;
}